### PR TITLE
Add Covid19-Tracker snppt

### PR DESCRIPTION
**Pull request by AnthonyLatty**
*Thanks for submitting a PR to the Snppts ✂️ library!*

## Submission description

Excited to showcase my Covid19 Tracker app, also added myself as author and submitted my first snippet 😃 💃

### CODE REVIEW

**Comment by sthewissen** [comment]
Nice! Would you mind reverting the changes on the csproj and removing the two files added by MFractor from the PR?

File: src/Snppts/Snppts.csproj [hunk 1]
```diff
@@ -6,7 +6,10 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.App" />
+    <PackageReference Include="Microsoft.AspNetCore.App">
```
Comment (jfversluis) [inline src/Snppts/Snppts.csproj]: Revert this please

File: src/Snppts/Snippets/Covid19TrackerUI.cs [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+        public bool ContainsiOSSample => true;
+
+        public bool ContainsUWPSample => false;
+
+        public IAmAnAuthor AuthorInfo => new MarcAnthonyLatty();
+
+        public IEnumerable<Uri> ImageUris => new List<Uri>
+        {
+            new Uri("https://github.com/AnthonyLatty/Covid19.Forms/blob/master/Artwork/artwork_droid_3.jpg"),
```
Comment (jfversluis) [inline src/Snppts/Snippets/Covid19TrackerUI.cs]: You'll want to use a URL like this `https://raw.githubusercontent.com/AnthonyLatty/Covid19.Forms/master/Artwork/artwork_droid_3.jpg` notice the "raw" in the first part. Else the image get redirected to a page on GitHub and they won't show

**Review by jfversluis — CHANGES_REQUESTED** [review]
Great stuff! A couple of small things and we're ready to add your first snippet!

File: src/Snppts/Snippets/Covid19TrackerUI.cs (hunk 2 above)
Comment (AnthonyLatty) [reply src/Snppts/Snippets/Covid19TrackerUI.cs]: Thanks!

File: src/Snppts/Snppts.csproj (hunk 1 above)
Comment (AnthonyLatty) [reply src/Snppts/Snppts.csproj]: Done..

**Review by jfversluis — APPROVED** [review]

**Pull request merged**